### PR TITLE
[FIX] open view corresponding to the type of invoice

### PR DESCRIPTION
**Pull request by hbrunn**
this is a backport of 28e98c5

### CODE REVIEW

File: account_invoice_merge/wizard/invoice_merge.py [hunk 1]
```diff
@@ -111,19 +111,18 @@ def merge_invoices(self, cr, uid, _ids, context=None):
...
+        invoices = invoice_obj.browse(cr, uid, ids, context=context)
+        xid = {
+            'out_invoice': 'action_invoice_tree1',
+            'out_refund': 'action_invoice_tree3',
+            'in_invoice': 'action_invoice_tree2',
+            'in_refund': 'action_invoice_tree4',
+        }[invoices[0].type]
+        action = aw_obj.for_xml_id(cr, uid, 'account', xid, context=context)
+        action.update({
+            'domain': [('id', 'in', ids + allinvoices.keys())],
```
Comment (pedrobaeza) [inline account_invoice_merge/wizard/invoice_merge.py:125]: Why preserve ids list?

Comment (hbrunn) [inline account_invoice_merge/wizard/invoice_merge.py:125]: i took that over from the original. Seems like a good idea to me, so you see the original invoices too and can check if everything went well

Comment (pedrobaeza) [inline account_invoice_merge/wizard/invoice_merge.py:125]: But this ids are removed after the merging, doesn't it?

Comment (hbrunn) [inline account_invoice_merge/wizard/invoice_merge.py:125]: no, cancelled

Comment (pedrobaeza) [inline account_invoice_merge/wizard/invoice_merge.py:125]: Ah, OK

**Comment by pedrobaeza** [comment]
:+1:

**Comment by sbidoul** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**